packages/testing/playwright/fixtures/base.ts

```diff
@@ -161,26 +185,106 @@ export const test = base.extend<
 		{ scope: 'worker' },
 	],
 
-	baseURL: async ({ n8nUrl, dbSetup }, use) => {
+	baseURL: async ({ frontendUrl, dbSetup }, use) => {
 		void dbSetup; // Ensure dbSetup runs first
-		await use(n8nUrl);
+		await use(frontendUrl);
 	},
 
-	n8n: async ({ context }, use, testInfo) => {
+	n8n: async ({ context, backendUrl, frontendUrl }, use, testInfo) => {
 		await setupDefaultInterceptors(context);
 		const page = await context.newPage();
-		const n8nInstance = new n8nPage(page);
+
+		// Only create a separate API context when backend and frontend URLs differ
+		const useSeparateApiContext = backendUrl !== frontendUrl;
+		let apiContext: APIRequestContext | undefined;
+		let api: ApiHelpers;
+
+		if (useSeparateApiContext) {
+			// Create a separate API context with backend URL for API calls
+			apiContext = await request.newContext({ baseURL: backendUrl });
+			api = new ApiHelpers(apiContext);
+		} else {
+			// Use the page's context when URLs are the same (default behavior)
+			api = new ApiHelpers(context.request);
+		}
+
+		const n8nInstance = new n8nPage(page, api);
 		await n8nInstance.api.setupFromTags(testInfo.tags);
+
+		// Default to owner authentication if no explicit auth tag is provided
+		// This maintains backward compatibility with tests that expect to be authenticated by default
+		const authContextToCheck = apiContext ?? context;
+		let apiCookies = await authContextToCheck.storageState();
+		let authCookie = apiCookies.cookies.find((cookie) => cookie.name === N8N_AUTH_COOKIE);
+
+		const hasExplicitNoAuth = testInfo.tags.some((tag) => tag.includes('@auth:none'));
```

Comment:
auth:none is expected to be unauthenticated.

If there is no tag, we sign in as the owner. Other than that we do what the tag says e.g member, none. To allow for testing of unauthenticated scenarios